src/math/min/index.js

```diff
@@ -0,0 +1,16 @@
+function min (array) {
+  if (!Array.isArray(array)) {
+    return undefined
+  }
+
+  if (!array.length > 0) {
+    return undefined
+  }
+
+  if (array.includes(NaN, undefined)) {
```

Comment:
Looks it is not enough to check. What if there are other "bad" values like null, 'string', false, ...? 
I think you should check whether every value is number and every value is not NaN. (because NaN - number)